bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/lib/device/connection/WifiConnection.java

```diff
@@ -120,7 +121,7 @@ public boolean connect() {
                 mDevice.setConnectionState(ConnectState.DISCONNECTED);
                 mSocket = null;
             }
-        });
+        }, "OH-binding-" + YeelightBindingConstants.BINDING_ID + "-WIFIConnection");
```

Comment:
Should it match class name?
```suggestion
        }, "OH-binding-" + YeelightBindingConstants.BINDING_ID + "-WifiConnection");
```

Reply:
See my comment above..

Reply:
But if it just the capitalisation I am fine with that too.